.github/workflows/serverless-build-push.yaml

```diff
@@ -0,0 +1,48 @@
+name: serverless build (push)
+
+on:
+  push:
+    branches: [ "main", "release-*" ]
+    paths-ignore:
+      - 'docs/**'
+      - 'examples/**'
+
+permissions:
+  id-token: write
+  contents: read
```

Comment:
I would suggest to keep comments about permissions ([source](https://github.com/kyma-project/test-infra/blob/main/cmd/image-builder/README.md#workflow-permissions))

Reply:
I will add it in next PR, thanks